src/_pytest/nodes.py

```diff
@@ -331,13 +332,19 @@ def _prunetraceback(self, excinfo):
         pass
 
     def _repr_failure_py(
-        self, excinfo: ExceptionInfo[Union[Failed, FixtureLookupError]], style=None
+        self,
+        excinfo: ExceptionInfo[
+            Union[Failed, FixtureLookupError, ConftestImportFailure]
+        ],
+        style=None,
     ) -> Union[str, ReprExceptionInfo, ExceptionChainRepr, FixtureLookupErrorRepr]:
         if isinstance(excinfo.value, fail.Exception):
             if not excinfo.value.pytrace:
                 return str(excinfo.value)
         if isinstance(excinfo.value, FixtureLookupError):
             return excinfo.value.formatrepr()
+        if isinstance(excinfo.value, ConftestImportFailure):
```

Comment:
This should probably come first, in case the wrapped exception is a `fail.Exception` or a `FixtureLookupError` (if that's possible -- but even if not, probably a good idea to move).

Reply:
done